web/modules/custom/rdf_entity/src/Entity/Rdf.php

```diff
@@ -325,6 +340,55 @@ public function setUnpublished() {
   /**
    * {@inheritdoc}
    */
+  public function getOwner() {
+    return ($this->hasUidMapping() ? $this->get('uid')->entity : NULL);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getOwnerId() {
+    return ($this->hasUidMapping() ? $this->getEntityKey('uid') : NULL);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setOwnerId($uid) {
+    if ($this->hasUidMapping()) {
+      $this->set('uid', $uid);
+    }
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setOwner(UserInterface $account) {
+    if ($this->hasUidMapping()) {
+      $this->set('uid', $account->id());
+    }
+    return $this;
+  }
+
+  /**
+   * Returns whether the bundle of the entity has a mapping for the uid key.
+   *
+   * @return bool
+   *    Whether the entity bundle has a value for the uid key mapping.
+   */
+  protected function hasUidMapping() {
+    if (empty($this->bundle())) {
+      return NULL;
+    }
+    $bundle = $this->entityTypeManager()->getStorage('rdf_type')->load($this->bundle());
```

Comment:
The bundle is statically cached after first load (this method can be called REALLY multiple times per request). After loading the bundle, the third party settings are in the loaded array! no request is being done directly in the config storage so it's fine I think. In the end, it is just an array seek.